File: java/program/Product.java
class Product {
    protected String idProduct;
    protected String name;
    protected String brand;
    protected double price;

    public Product(String idProduct, String name, String brand, double price) {
        this.idProduct = idProduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    public String getIdProduct() {
        return idProduct;
    }

    public String getName() {
        return name;
    }

    public String getBrand() {
        return brand;
    }

    public double getPrice() {
        return price;
    }

    public void setIdProduct (String idProduct) {
        this.idProduct = idProduct;
    }

    public void setName (String name) {
        this.name = name;
    }

    public void setBrand (String brand) {
        this.brand = brand;
    }

    public void setPrice (double price) {
        this.price = price;
    }
}

File: python/program/Product.py
class Product:
    def __init__(self, id_product, name, brand, price):
        self.id_product = id_product
        self.name = name
        self.brand = brand
        self.price = price

    def get_id_product(self):
        return self.id_product

    def get_name(self):
        return self.name

    def get_brand(self):
        return self.brand

    def get_price(self):
        return self.price

    def set_id_product(self, id_product):
        self.id_product = id_product

    def set_name(self, name):
        self.name = name

    def set_brand(self, brand):
        self.brand = brand

    def set_price(self, price):
        self.price = price